perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -0,0 +1,473 @@
+"""Python script for benchmarking listing operation.
+
+This python script benchmarks and compares the latency of listing operation in
+persistent disk vs GCS bucket. It creates the necessary directory structure,
+containing files and folders, needed to test the listing operation. Furthermore it can
+optionally upload the results of the test to a Google Sheet. It takes input a
+JSON config file whcih contains the info regrading directory structure and also through
+which multiple tests of different configurations can be performed in a single run.
+
+Typical usage example:
+  $ python3 listing_benchmark.py [-h] [--keep_files] [--upload] [--num_samples NUM_SAMPLES] [--message MESSAGE] --command COMMAND config_file
+
+  Flag -h: Typical help interface of the script.
+  Flag --keep_files: Do not delete the generated directory structure from the persistent disk after running the tests.
+  Flag --upload: Uploads the results of the test to the Google Sheet.
+  Flag --num_smaples NUM_SAMPLES: Runs each test for NUM_SAMPLES times.
+  Flag --message MESSAGE: Takes input a message string, which describes/titles the test.
+  Flag --command COMMAND (required): Takes a input a string, which is the command to run the tests on.
+  config_file (required): Path to the JSON config file which contains the details of the tests.
+
+Note: This python script is dependent on generate_files.py.
+"""
+
+import argparse
+import ast
+import configparser
+import json
+import logging
+import os
+import statistics as stat
+import subprocess
+import sys
+import time
+
+import directory_pb2 as directory_proto
+
+sys.path.insert(0, '..')
+import generate_files
+import numpy as np
+import texttable
+
+from google.protobuf.json_format import Parse, ParseDict
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s [%(levelname)s] %(message)s',
+    handlers=[logging.StreamHandler(sys.stdout)],
+)
+log = logging.getLogger()
+
+
+def OutputResults(folders, results_list, message, num_samples) -> dict:
```

Comment:
you are not outputting results. Call it parse results

Reply:
done